### PR TITLE
Design

### PR DESCRIPTION
**Pull request by danmainah**
### Math magicians: components

**In this milestone I did the following**

- Create a directory called components and inside it, a file called calculator.
- In Calculator.js, create a React component that matches the design given.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        </div>
+        <div className="row">
+          <button className="background col-6" type="submit">0</button>
+          <input className="background col-3" type="submit" value="." />
+          <input className="btn danger col-3" type="reset" value="=" />
+        </div>
+      </div>
+    </div>
+  </div>
+);
```
Comment (vichuge) [inline src/components/Calculator.js]: - [x] The Calculator.js component should be class-based. Please, fix this. 🐶

File: src/App.js [hunk 2]
```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```
Comment (vichuge) [inline src/App.js]: - [x] The `App.js` component should be class-based. Please, fix it too. 🦊

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [x] Please, delete all the unnecessary elements, in this case is all on the image:
image

| given template | your project |
| --- | --- |
| image | image |

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (danmainah) [reply src/components/Calculator.js]: Thanks for the review ,I have implemented changes requested

Comment (henatan99) [reply src/components/Calculator.js]: This issue is well addressed. Nice work! :clap:

File: src/App.js (hunk 2 above)
Comment (henatan99) [reply src/App.js]: This issue is well addressed. Great work. :+1: :clap:

Comment (danmainah) [reply src/App.js]: Thank you for the feedback henatan

File: src/App.css [hunk 3]
```diff
@@ -2,38 +2,27 @@
...
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+.text {
+  text-align: right;
 }
```
Comment (henatan99) [inline src/App.css]: - [x] OPTIONAL: Kindly make the text color similar ot the original. You should always stick to the given original design as if it is the client's requirement :pray: .

image

image

**Review by henatan99 — APPROVED** [review]
Hi @danmainah , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css (hunk 3 above)
Comment (danmainah) [reply src/App.css]: Thanks made the suggested changes

**Pull request merged**